tests/reporter/kubernetes.go

```diff
@@ -76,9 +78,37 @@ func (r *KubernetesReporter) SpecDidComplete(specSummary *types.SpecSummary) {
 	r.logEvents(virtCli, specSummary)
 	r.logPods(virtCli, specSummary)
 	r.logVMIs(virtCli, specSummary)
+	r.logDomainXMLs(virtCli, specSummary)
 	r.logLogs(virtCli, specSummary)
 }
 
+func (r *KubernetesReporter) logDomainXMLs(virtCli kubecli.KubevirtClient, specSummary *types.SpecSummary) {
+
+	f, err := os.OpenFile(filepath.Join(r.artifactsDir, fmt.Sprintf("%d_domains.log", r.failureCount)),
+		os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to open the file: %v", err)
+		return
+	}
+	defer f.Close()
+
+	vmis, err := virtCli.VirtualMachineInstance(v1.NamespaceAll).List(&v12.ListOptions{})
```

Comment:
v-twelve? typo I believe

Reply:
Ah that is autocomplete on import. I am sometimes lazy with renaming the import aliases to something useful. Will address it.

Reply:
this was not introduced with this PR, not worth waiting for a improvement. Will merge.